legal-api/src/legal_api/models/business.py

```diff
@@ -126,6 +126,15 @@ def delivery_address(self):
         return db.session.query(Address).filter(Address.business_id == self.id).\
             filter(Address.address_type == Address.DELIVERY)
 
+    @property
+    def good_standing(self):
+        """Return true if in good standing, otherwise false."""
+        last_ar = self.founding_date
+        if self.last_ar_date:
+            last_ar = self.last_ar_date
+
+        return last_ar + datedelta.datedelta(years=1, months=2, days=1) > datetime.now(timezone.utc)
```

Comment:
and switching to our library then makes it 
```
    > datetime.utcnow()
```

https://github.com/bcgov/lear/blob/master/legal-api/src/legal_api/utils/datetime.py#L25

Reply:
this is done